themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -139,13 +139,13 @@
       <a class="sr-only" href="#content"><?=$this->transEsc('Skip to content') ?></a>
       <?=$this->render('header.phtml')?>
     </header>
-    <nav class="breadcrumbs" aria-label="<?=$this->transEscAttr('Breadcrumbs') ?>">
-      <div class="container">
-        <?php if (!empty($this->layout()->showBreadcrumbs ?? true) && !empty($this->layout()->breadcrumbs)): ?>
-          <ul class="breadcrumb hidden-print"><?=$this->layout()->breadcrumbs ?></ul>
-        <?php endif; ?>
-      </div>
-    </nav>
+    <?php if ($this->layout()->showBreadcrumbs ?? true): ?>
+      <nav class="breadcrumbs" aria-label="<?=$this->transEscAttr('Breadcrumbs') ?>">
+        <div class="container">
+            <ul class="breadcrumb hidden-print"><?=$this->layout()->breadcrumbs ?></ul>
```

Comment:
I see that we're now unconditionally displaying this `<ul>` whether or not it has any contents. Is there value in having an empty list, or should we restore the `<?php if !empty($this->layout()->breadcrumbs)): ?>` check around this line?

If you think it is best to have the unconditional empty list, then this line should be de-indented by two spaces for consistency with the rest of the file.

Reply:
At first, I thought it could be useful during development to give a heads-up to the programmer about missing breadcrumbs, even if they were expected. But thinking about it now, it doesn't seem to make much sense because the rest of the code doesn't do that for other situations. What's your take on it?

Reply:
Yes, I think it's probably better to omit the empty container.